sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_enums.py

```diff
@@ -21,9 +21,9 @@ class KeyCurveName(with_metaclass(CaseInsensitiveEnumMeta, str, Enum)):
 class KeyExportEncryptionAlgorithm(with_metaclass(CaseInsensitiveEnumMeta, str, Enum)):
     """Supported algorithms for protecting exported key material"""
 
-    CKM_RSA_AES_KEY_WRAP = "CKM_RSA_AES_KEY_WRAP"
-    RSA_AES_KEY_WRAP_256 = "RSA_AES_KEY_WRAP_256"
-    RSA_AES_KEY_WRAP_384 = "RSA_AES_KEY_WRAP_384"
+    ckm_rsa_aes_key_wrap = "CKM_RSA_AES_KEY_WRAP"
```

Comment:
Why the change from upper case to lower case? I thought Python enums were supposed to be [capital letters](https://azure.github.io/azure-sdk/python_design.html#enumerations)

Reply:
They are! Unfortunately we had already GA-ed with all lower-case in all packages except for `azure-keyvault-administration`. Anna suggested that we keep enum values consistent within a package (so lower-case here), but that continuing with proper upper-casing in `administration` is preferred